packages/modules/mssqlserver/src/mssqlserver-container.ts

```diff
@@ -0,0 +1,123 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const MSSQL_PORT = 1433;
+
+type ProductKey = `${string}-${string}-${string}-${string}-${string}`;
+
+type Edition = "Evaluation" | "Developer" | "Express" | "Web" | "Standard" | "Enterprise" | ProductKey;
+
+const PASSWORD_CATEGORY_VALIDATION_PATTERNS = [/[A-Z]+/, /[a-z]+/, /[0-9]+/, /[@$!%*?&]+/];
+
+export class MSSQLServerContainer extends GenericContainer {
+  private database = "master";
+  private username = "SA";
+  private password = "Passw0rd";
+  private edition: Edition = "Developer";
```

Comment:
`Developer` is the default value for `MSSQL_PID`, so I don't think it is needed. Also, this can added `.withEnvironment({ "MSSQL_PID": "Express })` if required.